.github/dependabot.yml

```
@@ -0,0 +1,68 @@
+version: 2
+updates:
+  # Monitor Go dependencies for critical packages
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    # Group updates to reduce noise while maintaining visibility
+    groups:
+      # Core Filecoin dependencies
+      filecoin-core:
+        patterns:
+          - "github.com/filecoin-project/go-state-types"
+          - "github.com/filecoin-project/go-address"
+          - "github.com/filecoin-project/go-amt-ipld/*"
+          - "github.com/filecoin-project/go-bitfield"
+          - "github.com/filecoin-project/go-cbor-util"
+          - "github.com/filecoin-project/go-commp-utils/*"
+          - "github.com/filecoin-project/go-crypto"
+          - "github.com/filecoin-project/go-f3"
+          - "github.com/filecoin-project/go-fil-commcid"
+          - "github.com/filecoin-project/go-hamt-ipld/*"
+          - "github.com/filecoin-project/go-jsonrpc"
+          - "github.com/filecoin-project/go-padreader"
+          - "github.com/filecoin-project/go-paramfetch"
+          - "github.com/filecoin-project/go-statemachine"
+          - "github.com/filecoin-project/go-statestore"
+          - "github.com/filecoin-project/go-storedcounter"
+          - "github.com/filecoin-project/pubsub"
+          - "github.com/filecoin-project/test-vectors/*"
+          - "github.com/filecoin-project/specs-actors"
+          - "github.com/filecoin-project/specs-actors/*"
+      # IPFS ecosystem dependencies including boxo
+      ipfs-ecosystem:
+        patterns:
+          - "github.com/ipfs/boxo"
+          - "github.com/ipfs/*"
+      # libp2p ecosystem dependencies
+      libp2p-ecosystem:
+        patterns:
+          - "github.com/libp2p/go-libp2p"
+          - "github.com/libp2p/*"
+      # Multiformats dependencies
+      multiformats:
+        patterns:
+          - "github.com/multiformats/*"
+      # Web and networking dependencies
+      web-networking:
+        patterns:
+          - "github.com/gorilla/mux"
+          - "github.com/gorilla/websocket"
+      # Database and encoding dependencies
+      data-storage:
+        patterns:
+          - "github.com/mattn/go-sqlite3"
+          - "github.com/whyrusleeping/cbor-gen"
+      # Hardware and crypto dependencies
+      crypto-hardware:
+        patterns:
+          - "github.com/zondax/ledger-filecoin-go"
+          - "golang.org/x/crypto"
+    # Additional configuration to ensure maintainers can review
+    open-pull-requests-limit: 10
+    reviewers:
+      - "@filecoin-project/lotus-maintainers"
+    labels:
+      - "dependencies"
+      - "automated"
```

```suggestion
      - "dependencies"
```

Removing since I don't see an `automated` label in this repo.